### ISSUE TITLE
4 failed tests due to a new connector HM

### ISSUE THREAD

**Issue by ampli** [opencog/link-grammar#1165]
Is this intentional?
``` text
linkparser> 
	Linkage 3, cost vector = (UNUSED=0 DIS= 0.00 LEN=6)

    +----->WV----->+----Ost---+
    +-->Wd---+-Ss*b+>HM>+Ds**c+
    |        |     |    |     |
LEFT-WALL this.p is.v   a  test.n
```

**Comment by linas** [comment]
No... but its an interesting mistake! Because its kind-of-ish correct ...

**Comment by linas** [comment]
fixed in commit 0bcc0b64f49bea7f
